### ISSUE TITLE
Underline on hover

### ISSUE THREAD

**Issue by tom-barbereau** [LordMathis/hugo-theme-nightfall#31]
Thanks for this great theme. I adapted it and it works great! 

Only encountered a minor issue that I tried to fix without any luck: have links underlined when hovered over. 

In the global_scss, as per the original theme, I have: 

```
a {
  color: $blue;
}
```

To activate the hover, it should be: 

```
a {
  color: $blue;
  text-decoration:none;
}

a:hover {
   text-decoration:underline;
}
```

Unfortunately, this doesnt do it... I tried also to take away the color: part, still doesnt work... Ideally, I'd like to have color (permanently) + underline (on hover).

Any suggestions how to work this one out? I am new to html/css so apologies if this is an obvious one that I just do not see.

**Comment by LordMathis** [comment]
The underline settings that you need to change are in `components/_post.scss`. There is 

```
...
a,
a:link,
a:visited,
a:hover,
a:active{
  text-decoration: none;
}
```

Take out the a:hover and create a new rule with underline for it.  This will however also underline the post excerpt in the post list page which is not ideal.

I think having normal links underlined on hover is a good idea, but I don't want to also underline the entire post excerpt.

**Comment by LordMathis** [comment]
Actually just fixed it in #34. Unless you wanted to enable underline for all links. Then I might have made it more difficult for you
